### PR TITLE
feat: adds password show hide toggle switch

### PR DESCRIPTION
**Pull request by rtpa25**
- closes https://github.com/hyperdxio/hyperdx/issues/39
- https://www.loom.com/share/33e09bd9e4c84368b33c1814bb2ffeaa {No more scroll on auth screen --> toggle switch on password input field}

### CODE REVIEW

**Comment by rtpa25** [comment]
@wrn14897 @MikeShi42 let me know if you need any more UI improvements on this page

**Comment by wrn14897** [comment]
> @wrn14897 @MikeShi42 let me know if you need any more UI improvements on this page

@rtpa25 Could you merge main and resolve the conflict ? I think we should be ready to merge this PR

File: packages/app/src/SVGIcons.tsx [hunk 1]
```diff
@@ -248,3 +248,36 @@ export function S3Icon({ style, width }: IconProps) {
     </svg>
   );
 }
+
+export function Eye({ style, width }: IconProps) {
```
Comment (wrn14897) [inline packages/app/src/SVGIcons.tsx:252]: I think we can use bootstrap icons instead https://icons.getbootstrap.com/?q=eye

**Comment by wrn14897** [comment]
This is done

**Pull request closed without merging**